### DISCUSSION TITLE
Why can’t I open the page using docker installation or local installation, and no error is reported in the log?

### DISCUSSION THREAD

**Discussion by Seapein** [devlive-community/datacap#699] [Q&A]
![Uploading 微信图片_20240307161915.png…]()
![微信图片_20240307161912](https://github.com/devlive-community/datacap/assets/37824176/1edc1fbf-506b-4193-99a6-59264f343610)
![微信图片_20240307161909](https://github.com/devlive-community/datacap/assets/37824176/2be5406d-db6d-4675-b438-f96eedaf08cf)
![微信图片_20240307161904](https://github.com/devlive-community/datacap/assets/37824176/828582da-e64c-4bc5-8c1c-69a59766bc43)

docker run -d -p 39096:9096 -v /opt/datacapdata/application.properties:/opt/app/datacap/configure/application.properties --name datacap qianmoq/datacap

**Comment by qianmoQ** [comment]
Message that will be displayed on users' first issue

[a comment by an automated account is not shown]

**Comment by Seapein** [comment]
![abc](https://github.com/devlive-community/datacap/assets/37824176/9d64bf2e-293a-4117-aae0-04542580dd9e)

[a comment by an automated account is not shown]

**Comment by Seapein** [comment]
![1a58c96020f504f11ed5eec1f7496ab](https://github.com/devlive-community/datacap/assets/37824176/2afc2c31-cc33-481a-9f27-e19dd6d083e7)

**Comment by qianmoQ** [comment]
> ![1a58c96020f504f11ed5eec1f7496ab](https://private-user-images.githubusercontent.com/37824176/310805954-2afc2c31-cc33-481a-9f27-e19dd6d083e7.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDk4MDM0MTEsIm5iZiI6MTcwOTgwMzExMSwicGF0aCI6Ii8zNzgyNDE3Ni8zMTA4MDU5NTQtMmFmYzJjMzEtY2MzMy00ODFhLTlmMjctZTE5ZGQ2ZDA4M2U3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMDclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzA3VDA5MTgzMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQyNzJmNDQ0NmYyYjQ4YzAxODliMGQwNTM0MjFjMjAwYWQ2ZTc2OWRmODZlODVjNDU3NTQ5MjBhZWI2MTQzNmImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.LSnLwlgfji2EyZ-rW9ubKWb7W31BLSjqvU3MiSJOVKc)

请使用 `docker logs datacap` 发一下具体的日志
Please use `docker logs datacap` to send specific logs

**Comment by Seapein** [comment]
> > ![1a58c96020f504f11ed5eec1f7496ab](https://private-user-images.githubusercontent.com/37824176/310805954-2afc2c31-cc33-481a-9f27-e19dd6d083e7.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDk4MDM0MTEsIm5iZiI6MTcwOTgwMzExMSwicGF0aCI6Ii8zNzgyNDE3Ni8zMTA4MDU5NTQtMmFmYzJjMzEtY2MzMy00ODFhLTlmMjctZTE5ZGQ2ZDA4M2U3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMDclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzA3VDA5MTgzMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQyNzJmNDQ0NmYyYjQ4YzAxODliMGQwNTM0MjFjMjAwYWQ2ZTc2OWRmODZlODVjNDU3NTQ5MjBhZWI2MTQzNmImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.LSnLwlgfji2EyZ-rW9ubKWb7W31BLSjqvU3MiSJOVKc)
> 
> 请使用 `docker logs datacap` 发一下具体的日志 Please use `docker logs datacap` to send specific logs

Hi,I reused docker deployment. The following is my deployment log. For the configuration file, I only modified the mysql related configuration. This time the log did not report an error, but the http://192.168.1.104:9096/ page still cannot be opened.

![1709860337484](https://github.com/devlive-community/datacap/assets/37824176/135bd5ba-48b1-4c16-b0e6-a892a0216b79)
Below is the command：
docker run -d -p 9096:9096 -v /opt/datacapdata/application.properties:/opt/app/datacap/configure/application.properties --name datacap qianmoq/datacap
![image](https://github.com/devlive-community/datacap/assets/37824176/7f2a583e-47c9-4551-a8d5-b58a1b4480f0)
Below is the configuration file：
![1709860654126](https://github.com/devlive-community/datacap/assets/37824176/44acdacd-2bc4-4bad-b153-3e4bc380fce7)
![1709860745997](https://github.com/devlive-community/datacap/assets/37824176/7360c052-224d-430f-9574-5ac36d1c5ffd)

**Comment by qianmoQ** [comment]
try `http://localhost:9096/ `

**Comment by Seapein** [comment]
> try `http://localhost:9096/ `
Hi,
This address doesn’t work either, because I deployed the service on the server 192.168.1.104

**Comment by qianmoQ** [comment]
https://answer.devlive.org/questions/10010000000000037/127-0-0-1-9096-neng-fang-wen-dao-xiu-gai-cheng-ip-de-zhi-jiu-wu-fa-fang-wen
